lib/annotate/annotate_models.rb

```diff
@@ -957,6 +957,7 @@ def map_col_type_to_ruby_classes(col_type)
     when 'datetime', 'timestamp', 'time'                 then Time.to_s
     when 'date'                                          then Date.to_s
     when 'text', 'string', 'binary', 'inet', 'uuid'      then String.to_s
+    when 'json', 'jsonb'                                 then Hash.to_s
```

Comment:
Can you elaborate on this?

Reply:
When a model has a json/jsonb column, it's transformed in Hash class in ActiveRecord. That's the same logic i've followed here.